### ISSUE TITLE
Express-Rescue 1.1.27 Crashing

### ISSUE THREAD

**Issue by EddyeBoy27** [rwillians/express-rescue#12]
Hello guys, I'm new in programming, and because this I don't know why the new version is crashing, I had to use specific version 1.1.26 to continue using without error.

```
internal/modules/cjs/loader.js:326
      throw err;
      ^

Error: Cannot find module '/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/node_modules/express-rescue/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:318:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:679:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:949:27)
    at Function.Module._load (internal/modules/cjs/loader.js:838:27)
    at Module.require (internal/modules/cjs/loader.js:1022:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/controllers/productController.js:1:16)
    at Module._compile (internal/modules/cjs/loader.js:1118:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)
    at Module.load (internal/modules/cjs/loader.js:982:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/node_modules/express-rescue/package.json',
  requestPath: 'express-rescue'
}
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by rwillians** [comment]
Thanks @EddyeBoy27, I'm taking a look at it right now

**Comment by rwillians** [comment]
Hey @EddyeBoy27, thanks for pointing that out. The problem has been fixed [here](https://github.com/rwillians/express-rescue/commit/6d508469fc47b4f47456bf38ad865e7b812e6a4b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R20).
A new versions has just been released, you should be able to use `v1.1.28` now.

**Comment by EddyeBoy27** [comment]
you're welcome @rwillians , I updated here but the issue continue, look:

```
internal/modules/cjs/loader.js:326
      throw err;
      ^

Error: Cannot find module '/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/node_modules/express-rescue/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:318:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:679:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:949:27)
    at Function.Module._load (internal/modules/cjs/loader.js:838:27)
    at Module.require (internal/modules/cjs/loader.js:1022:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/controllers/productController.js:1:16)
    at Module._compile (internal/modules/cjs/loader.js:1118:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)
    at Module.load (internal/modules/cjs/loader.js:982:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/media/liipe/HD_TB/Liipe_Xubuntu/mythings/NOTEBOOK/Projetos/sd-02-project-store-manager/node_modules/express-rescue/package.json',
  requestPath: 'express-rescue'
}
[nodemon] app crashed - waiting for file changes before starting...
```
